.github/workflows/update-helm-repo.yaml

```diff
@@ -145,7 +146,7 @@ jobs:
       - name: Install Helm
         uses: azure/setup-helm@v4
         with:
-          version: v3.5.2
+          version: v3.16.2
```

Comment:
Note that `helm push` was introduced in [`3.7.0`, which was release in 2021](https://github.com/helm/helm/releases/tag/v3.7.0).

I propose we update to the latest release, which is 3.16, as of today.